dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResourceHelper.java

```diff
@@ -70,6 +76,20 @@ public void setContainerLanguage(final Container container, final HttpServletReq
                 }
             }
         }
-    } // setContainerLanguage.
+    }
+
+    public ContainerWithStructuresView toResponseEntityContainerWithStructuresView(final Container container, final List<ContainerStructure> containerStructures) {
+        return new ContainerWithStructuresView(container, containerStructures);
+    }
 
+    public String getObjectToJsonString(final Object object) {
+        ObjectMapper mapper = DotObjectMapperProvider.getInstance().getDefaultObjectMapper();
+        try {
+            final String json = mapper.writeValueAsString(object);
```

Comment:
You can use

final String json = Try.of(()->DotObjectMapperProvider.getInstance().getDefaultObjectMapper().writeValueAsString(object)).getOrElse(StringPool.BLANK);

the lib is very powerful

Reply:
Code has been modified as per feedback/suggestions @jdotcms @freddyucv  please have a look again - thanks